frontend/lib/error/index.ts

```diff
@@ -1,12 +1,32 @@
+import { type NuxtError } from 'nuxt/app'
+
 export enum Remediation {
   None = 'none',
   Reload = 'reload',
   FileBug = 'file-bug',
   CheckUrl = 'check-url',
```

Comment:
maybe let's make these more human readable now that we're showing them to humans?

Reply:
We still aren't showing these to humans, we'll just use them to select the actual thing we display in the error modal